eldritch/events.py

```diff
@@ -3247,8 +3259,9 @@ def resolve(self, state):
       annotations = ["Done" if self.done[idx] else None for idx, _ in present_monsters]
       prompt = "Choose a monster to fight or evade"
       none_choice = "Ignore All" if self.auto_evade else None
-      self.choice = MonsterChoice(self.character, prompt, shown_monsters, annotations, none_choice)
-      state.event_stack.append(self.choice)
+      if shown_monsters:
```

Comment:
If shown_monsters has nothing, shouldn't all(self.done) be true? I guess I'm worried about a situation where we might show no monsters in the choice, but the event won't think it's "done" yet.